fixtures/attribute-behavior/package.json

```diff
@@ -10,6 +10,12 @@
     "react-scripts": "1.0.11",
     "react-virtualized": "^9.9.0"
   },
+  "devDependencies": {
+    "@playwright/test": "^1.41.2"
+  },
+  "optionalDependencies": {
+    "fsevents": "1.2.13"
```

Comment:
Required to work around https://github.com/yarnpkg/yarn/pull/7273 on Linux.

Also requires running install with `--ignore-optional`